tests/Integration/HeaderTest.php

```diff
@@ -155,6 +155,12 @@ public function testGet(): void
         $this->assertTrue($header->get('Font') instanceof Page);
         $this->assertTrue($header->get('Image') instanceof ElementMissing);
         $this->assertTrue($header->get('Resources') instanceof ElementMissing);
+
+        $content = '<</Type/Page/SubType//Text>>foo';
+        $position = 0;
+        $header = Header::parse($content, $document, $position);
+
+        $this->assertTrue($header->get('SubType') instanceof ElementMissing);
```

Comment:
Thank you for your latest changes.

Can you please add a short comment here, which describes what these lines do. I assume you try to exploit the error before but with your patch it works now?

Reply:
Yeah, it looks great, go for it.

Reply:
You might have to use `/*` instead of `/**`.